src/mame/machine/genfruit.h

```diff
@@ -0,0 +1,33 @@
+// license:BSD-3-Clause
+// copyright-holders:SomeRandomGuyIdk
+#ifndef GENFRUIT_H_
+#define GENFRUIT_H_
```

Comment:
This doesn’t follow the format for include guards.

Reply:
Fixed.